test/arrays.spec.ts

```diff
@@ -182,3 +181,35 @@ describe('coercion of array to scalar', () => {
   })
 })
 
+describe('range interpolation', () => {
+  it('with function', () => {
+    const engine = HyperFormula.buildFromArray([[0, 1, 2], ['=EXP(A1:C1)']])
+    expect(engine.getCellValue(adr('A2'))).toEqual(1)
+  })
+
+  it('with binary op', () => {
+    const engine = HyperFormula.buildFromArray([[0, 1, 2], [undefined, '=A1:C1+A1:C1']])
+    expect(engine.getCellValue(adr('B2'))).toEqual(2)
```

Comment:
I don't get the `undefined`s in those tests. Why do we add complexity and shift the formula instead keeping it in A2?

Reply:
to test that we are able to extract something from the middle of the range. Otherwise how do we know that we are not getting things by default from top-left corner of the range, always.

Reply:
ok 👍